src/Startup.cs

```diff
@@ -42,6 +42,15 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddControllers().AddNewtonsoftJson();
+
+            // Configure app insight if the key exists. New Azure regions require the use of connection strings
+            // instead of instrumentation keys.
+            string? appInsightConnectionString = _configuration["APPLICATIONINSIGHTS_CONNECTION_STRING"];
+            if (!string.IsNullOrWhiteSpace(appInsightConnectionString))
```

Comment:
do we use it for this PR?

Reply:
Yes, I already set application insight on our bot app service